### PR TITLE
Create foreman-nulldb package

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: debian/bionic/foreman/control [hunk 1]
```diff
@@ -54,15 +54,15 @@ Description: metapackage providing PostgreSQL dependencies for Foreman
  .
  The foreman package should be installed along with this package.
 
-Package: foreman-sqlite3
+Package: foreman-nulldb
```
Comment (ekohl) [inline debian/bionic/foreman/control:57]: I feel that we should mirror these subpackage names between RPM and debs. In RPM we have `foreman-build`. Would it make sense to do the same on debs?

Comment (mmoll) [reply debian/bionic/foreman/control]: Sure :heavy_check_mark:

Comment (ekohl) [reply debian/bionic/foreman/control]: @theforeman/packaging this is correct, right?

**Comment by evgeni** [comment]
The f-build RPM contains build macros and stuff like that. The nulldb bundler file and schema is in foreman-plugin which is "this is what you need installed to build a plugin".

**Comment by mmoll** [comment]
I think all that doesn't apply to DEB, so I'm inclined to name it "foreman-nulldb" again... Any strong arguments against this?

The plugin build seems to work just fine.

**Comment by ekohl** [comment]
In that case I think `foreman-nulldb` is fine.

**Pull request merged**